joern-cli/frontends/jimple2cpg/src/main/scala/io/joern/jimple2cpg/Jimple2Cpg.scala

```diff
@@ -74,7 +69,6 @@ class Jimple2Cpg {
       }
 
       configureSoot()
-      val cpg             = newEmptyCpg(outputPath)
       val metaDataKeyPool = new IntervalKeyPool(1, 100)
```

Comment:
Do we need these key pools?  It's only Plume that needs these. Can remove if it's within the scope of this PR.

Reply:
Done